application/event_sender/event_sender.py

```diff
@@ -78,23 +81,25 @@ def lambda_handler(event: ChangeRequestQueueItem, context: LambdaContext, metric
             # the message is DLQ'd - 5 times, if we can fix that then these message could be sent to the dlq
             # and deleted to avoid circuit breaking and even replaying when we know it will fail again
             if response.status_code >= 500 or response.status_code == 429:
-                logger.info("Potentially recoverabele breaking circuit to retry shortly")
+                logger.info("Potentially recoverable breaking circuit to retry shortly")
                 put_circuit_status(environ["CIRCUIT"], True)
             elif 400 <= response.status_code < 500:
-                logger.info("Permanent error sending to DLQ as no point retrying")
-                # TODO : Send message to DLQ
-                # sqs.send_message(
-                #     QueueUrl=environ["CR_DLQ_URL"],
-                #     MessageBody=event["change_request"],
-                #     # "MessageDeduplicationId": f"{sequence_number}-{hashed_payload}",
-                #     # "MessageGroupId": f"{change_request.service_id}",
-                #     MessageAttributes={
-                #         "correlation_id": {"DataType": "String", "StringValue": logger.get_correlation_id()},
-                #         "message_received": {"DataType": "Number", "StringValue": str(message_received)},
-                #         "dynamo_record_id": {"DataType": "String", "StringValue": record_id},
-                #         "ods_code": {"DataType": "String", "StringValue": self.nhs_entity.odscode},
-                #     },
-                #      )
+                logger.info("Permanent error sending to DLQ, Not retrying")
+                hashed_payload = sha256(str(event["change_request"]).encode()).hexdigest()
+                sqs.send_message(
+                    QueueUrl=environ["CR_DLQ_URL"],
+                    MessageBody=dumps(event["change_request"]),
+                    MessageDeduplicationId=f"{time_ns()}-{hashed_payload}",
+                    MessageGroupId=odscode,
```

Comment:
have used serviceId here see event_processor that puts the original message on the queue

Reply:
i.e. add  MessageDeduplicationId and MessageGroupId to the metadata